docs/howtos/recipes_ibkx_partition.md

```diff
@@ -110,16 +110,16 @@ server.
 First step is to create new table in which data will be restored
 
 ```sql
-> mysql> CREATE TABLE name_p4 (
-> id int(11) NOT NULL AUTO_INCREMENT,
-> name text NOT NULL,
-> imdb_index varchar(12) DEFAULT NULL,
-> imdb_id int(11) DEFAULT NULL,
-> name_pcode_cf varchar(5) DEFAULT NULL,
-> name_pcode_nf varchar(5) DEFAULT NULL,
-> surname_pcode varchar(5) DEFAULT NULL,
-> PRIMARY KEY (id)
-> ) ENGINE=InnoDB AUTO_INCREMENT=2812744 DEFAULT CHARSET=utf8
+mysql> CREATE TABLE name_p4 (
+id int(11) NOT NULL AUTO_INCREMENT,
+name text NOT NULL,
+imdb_index varchar(12) DEFAULT NULL,
+imdb_id int(11) DEFAULT NULL,
+name_pcode_cf varchar(5) DEFAULT NULL,
+name_pcode_nf varchar(5) DEFAULT NULL,
+surname_pcode varchar(5) DEFAULT NULL,
+PRIMARY KEY (id)
+) ENGINE=InnoDB AUTO_INCREMENT=2812744 DEFAULT CHARSET=utf8
```

Comment:
There are references to .exp at line 132, 136. Please check

Reply:
Updated